subxt/Cargo.toml

```diff
@@ -40,6 +40,16 @@ jsonrpsee-web = ["jsonrpsee/async-wasm-client", "jsonrpsee/client-web-transport"
 # latest features exposed by the metadata.
 unstable-metadata = []
 
+# Activate this to expose the Light Client functionality.
+# Note that this feature is experimental and things may break or not work as expected.
+unstable-light-client = [
```

Comment:
I'm think exposing a feature flag `unstable` which has all unstable stuff i.e, both metadata, light client and other stuff would be better.

Reply:
I think keeping them separate prob makes sense here, because `unstable-metadata` doesn't just add something but also changes how Subxt fetches metadata from a node, so one may plausibly want to try light clients without also having subxt try to fetch V15 metadata :)